Kodilla-testing/src/test/java/com/kodilla/testing/shape/ShapeCollectorTestSuite.java

```diff
@@ -1,8 +1,86 @@
 package com.kodilla.testing.shape;
-
+import com.kodilla.testing.shape.ShapeCollector;
+import org.junit.*;
+import java.util.ArrayList;
+import java.util.List;
 
 public class ShapeCollectorTestSuite {
 
+    private static int testCounter = 0;
+    public ShapeCollector shapeCollector = new ShapeCollector();
+    private List<Shape> figureList = new ArrayList<>();
+
+    @BeforeClass
+    public static void beforeAllTests() {
+
+        System.out.println("This is the beginning of tests.");
+    }
+
+    @AfterClass
+    public static void afterAllTests() {
+        System.out.println("All tests are finished.");
+    }
+
+    @Before
+    public void beforeEveryTest() {
+        testCounter++;
+        System.out.println("Preparing to execute test #" + testCounter);
+    }
+
+    @Test
+    public void testAddFigure() {
+        //Given
+        Shape shape1 = new Circle("circle", 10);
+        Shape shape2 = new Triangle("triangle", 10, 12);
+        Shape shape3 = new Square("square", 10);
+        //When
+        shapeCollector.addFigure(shape1);
+        shapeCollector.addFigure(shape2);
+        shapeCollector.addFigure(shape3);
+        //Then
+        Assert.assertEquals(3, figureList.size());
```

Comment:
`figureList` jest polem tej klasy. Operacja `shapeCollector.addFigure()` dodaje figurę do listy wewnątrz `shapeCollectora` a nie do tej listy `figureList` przecież

Reply:
No ok, tylko, że chcę wywołać w teście metodę addFigure(), która jest już we właściwej klasie, czuli w class ShapeCollector{} a nie class ShapeCollectorTestSuite{}